src/OrchardCore/OrchardCore.Users.Core/Services/UserStore.cs

```diff
@@ -462,13 +462,14 @@ public async Task AddToRoleAsync(IUser user, string normalizedRoleName, Cancella
             if (user is User u)
             {
                 var roleNames = await _roleService.GetRoleNamesAsync();
+                var roleName = roleNames?.FirstOrDefault(r => NormalizeKey(r) == normalizedRoleName);
```

Comment:
```suggestion
                var roleName = roleNames.FirstOrDefault(r => NormalizeKey(r) == normalizedRoleName);
```

Reply:
Will check, for now here it is as it was before

Reply:
Done locally